rfd/0147-repo-fork-syncing.md

```diff
@@ -0,0 +1,60 @@
+---
+authors: Mike Jensen (mike.jensen@goteleport.com)
+state: draft
+---
+ 
+# RFD 144 - Automatic Repo Syncing
+ 
+## What
+ 
+This RFD describes how we can sync a repo continuously and in an automatic way.  This can be used to provide private copies of a repo.
+ 
+## Why
+ 
+We currently face a few frictions which could be addressed through a more robust repo sync mechanism:
+1. Our internal `teleport-private` fork used for security development must be manually updated
```

Comment:
Is the RFD solely concerned about teleport-private, or it applicable to any forks? Could we make that clearer here and/or in the title, depending on which we mean?

Reply:
`teleport-private` is the immediate need, but I am trying to make this as flexible as possible for future needs.

Reply:
I have clarified the scope, let me know if you have more questions